giddy-banana/routes/authRouter.js

```diff
@@ -0,0 +1,7 @@
+import { Router } from "express";
+import { refreshTokenRouter } from "./refreshTokenRoute.js";
```

Comment:
Remove curly braces here, because 
```SyntaxError: The requested module './refreshTokenRoute.js' does not provide an export named 'refreshTokenRouter'```